templates/search_results.hbs

```diff
@@ -41,7 +41,7 @@
           </ul>
         </section>
       {{/if}}
-      {{#if help_center.community_enabled}}
+      {{#if type_filters}}
```

Comment:
Do you know how truthiness works in CurlyBars? Will this return true if `type_filters` is an empty array?

Reply:
Answered my own question, an empty collection [resolves to false](https://github.com/zendesk/curlybars/blob/master/docs/templates.md#how-conditions-are-evaluated).

Reply:
We do the same for `source_filters` up above 😸